File: Unidad_III/Felipe/Java/src/HelloClass.java
public class HelloClass
{
	public static void main(String []args) 
	{
		String mensaje = "Hello there!";

		//Inicializar objeto
    		HolaMundo Obj = new HolaMundo(mensaje);

		//Imprimir mensaje
		Obj.say_hello();
	}
		
}

File: Unidad_III/master/C++/src/HelloClass.cpp
#include <stdio.h>
#include <stdio.h>
#include "HolaMundo.hpp"

//Main
int main()
{

  //Variables locales
  std::string mensaje = "Hello there!";

  //Inicializar objetos
  HolaMundo* Obj = new HolaMundo(mensaje);

  //Imprimir mensaje
  Obj->say_hello();

return 0;
}
